### PR TITLE
remove craueconfig domain name setting and add a proper one in parame…

### PR DESCRIPTION
**Pull request by tcitworld**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | https://github.com/wallabag/doc/pull/8
| Translation   | no
| Fixed tickets | #...
| License       | MIT

Replace the craue config setting with a dedicated parameter for domain name.

### CODE REVIEW

**Comment by j0k3r** [comment]
Do not forget the `InstallCommand`.

**Comment by tcitworld** [comment]
What is it more to handle ?

**Comment by j0k3r** [comment]
I don't know.

File: src/Wallabag/CoreBundle/Resources/config/services.yml [hunk 1]
```diff
@@ -195,7 +195,7 @@ services:
         arguments:
             - "@wallabag_core.entry.download_images.client"
             - "%kernel.root_dir%/../web/assets/images"
-            - '@=service(''craue_config'').get(''wallabag_url'')'
+            - '%wallabag_core.domain_name%'
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/config/services.yml]: Be consistent, use `%domain_name%` instead.

File: app/DoctrineMigrations/Version20170606155640.php [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Add api_user_registration in craue_config_setting.
```
Comment (j0k3r) [inline app/DoctrineMigrations/Version20170606155640.php]: :no_good_man:

File: app/config/parameters.yml.dist [hunk 3]
```diff
@@ -23,6 +23,8 @@ parameters:
     # with PostgreSQL and SQLite, you must set "utf8"
     database_charset: utf8mb4
 
+    domain_name: wallabag.me
```
Comment (j0k3r) [inline app/config/parameters.yml.dist]: Shouldn't this start by `http://`?

File: app/config/parameters_test.yml [hunk 4]
```diff
@@ -8,3 +8,4 @@ parameters:
     test_database_path: "%env(TEST_DATABASE_PATH)%"
     env(TEST_DATABASE_PATH): "%kernel.root_dir%/../data/db/wallabag_test.sqlite"
     test_database_charset: utf8
+    domain_name: 'http://v2.wallabag.org'
```
Comment (j0k3r) [inline app/config/parameters_test.yml]: Looks like test still use `wallabag.me`: https://travis-ci.org/wallabag/wallabag/jobs/239983368#L661

File: app/config/parameters.yml.dist (hunk 3 above)
Comment (tcitworld) [inline app/config/parameters.yml.dist]: yes

File: app/config/parameters_test.yml (hunk 4 above)
Comment (tcitworld) [inline app/config/parameters_test.yml]: Hmm. Bothering.

File: app/config/wallabag.yml [hunk 5]
```diff
@@ -26,6 +26,7 @@ wallabag_core:
     fetching_error_message: |
         wallabag can't retrieve contents for this article. Please <a href="http://doc.wallabag.org/en/user/errors_during_fetching.html#how-can-i-help-to-fix-that">troubleshoot this issue</a>.
     api_limit_mass_actions: 10
+    domain_name: '%domain_name%'
```
Comment (j0k3r) [inline app/config/wallabag.yml]: Also, I don't think you need to include the domain here. We can use it from the parameter directly.

File: app/config/parameters.yml.dist [hunk 6]
```diff
@@ -23,7 +23,7 @@ parameters:
     # with PostgreSQL and SQLite, you must set "utf8"
     database_charset: utf8mb4
 
-    domain_name: wallabag.me
+    domain_name: http://your-wallabag-url-instance.com
```
Comment (nicosomb) [inline app/config/parameters.yml.dist]: we can promote `https` here.

**Pull request merged**